frontend/tailwind.config.js

```diff
@@ -32,9 +32,9 @@ export default {
         'dark-theme-background': '#092e40',
         'dark-theme-foreground': '#25a5e3',
       },
-      fontFamily: {
-        sans: ['Poppins', ...defaultTheme.fontFamily.sans],
-      },
+      // fontFamily: {
+      //   sans: ['Poppins', ...defaultTheme.fontFamily.sans],
+      // },
     },
```

Comment:
same

Reply:
I couldnt find the use Poppins and Lighthouse showed that font is one of the reasons for performance decrease, so commented it out for the time being.

Reply:
It is not being used? Thats interesting ... okay... But i actually don't see any visible changes with respect to font, i remember we had removed hardcoded value and placed it here, thought it is taking default from here, but now I am even confused, can you once cross verify if add and remove of this, is the website ka font changing. I couldn't actually see any change, 

And if that is the case, can you see the way to add it, may be all these days Poppins was never added , which is supposed to be the application's default font

Reply:
ill check and get back to you